spec/checks/at_least_one_file_check_spec.rb

```diff
@@ -0,0 +1,59 @@
+require_relative '../spec_helper'
+
+describe RepoAudit::Checks::AtLeastOneFileCheck do
+  let(:metadata)  { {description: 'check description', style_guide_url: 'www.example.com'} }
+  let(:arguments) { {filenames: %w[file1.txt file2.txt]} }
+
+  subject { described_class.new(metadata, arguments) }
+
+  it_behaves_like 'a Check', name: :at_least_one_file_check
+
+  context '#run' do
+    let(:repository) { double('Repository', full_name: 'org/repo-name') }
+
+    context 'for a success result in the first file' do
+      it 'returns the expected result' do
```

Comment:
Bit late now, as this has been merged, but the `context` block here seems a bit redundant? Saying `it 'returns the expected result'` tells the reader basically nothing about the test…